agu_oss/clean_data.py

```diff
@@ -19,6 +19,9 @@ def open_and_clean(url):
 
     """
 
+    print("Starting data cleanup.")
```

Comment:
nice addition to the code @jschap1 ! looks like your pull  request is building just fine!